packages/patrol_cli/lib/src/crossplatform/app_options.dart

```diff
@@ -11,12 +11,14 @@ class FlutterAppOptions {
     required this.flavor,
     required this.buildMode,
     required this.dartDefines,
+    this.dartDefineFromFile = const {},
```

Comment:
Let's make it `required`. This way we'll see all the places where it should be passed but currently isn't (for example [here](https://github.com/leancodepl/patrol/blob/f0bae1d5b2cb19d451d9f83ef83789f28ab4e5f3/packages/patrol_cli/lib/src/commands/develop.dart#L234-L239)).

Reply:
got you! makes sense